frontend/src/components/eventPage/DisplayEventInfo.jsx

```diff
@@ -15,14 +15,17 @@ const DisplayEventInfo = ({ event, user }) => {
     if (user != null && user != undefined) Logged = true
 
     useEffect(() => {
-        fetchUserAttendance()
+        if (Logged) fetchUserAttendance()
     }, [userIsAttending])
 
     const handleRegister = async () => {
         if (userIsAttending == 0) {
+            console.log(
+                `${url}/attendance/user/${user.username}/${current_eventId}`
+            )
             try {
                 await axios.post(
-                    `${url}/attendance/user/${user.id}/${current_eventId}`,
+                    `${url}/attendance/user/${user.username}/${current_eventId}`,
```

Comment:
https://github.com/CristobalMe/Event_Finder/pull/17#discussion_r1671583870

Reply:
To do